### PR TITLE
Boilerplate Spec for master Branch

### PR DESCRIPTION
**Pull request by dblevins**
Generated boilerplate spec.  Feel free to merge, then refine.

 See https://wiki.eclipse.org/How_to_Prepare_API_Projects_to_Jakarta_EE_8_Release

### CODE REVIEW

File: api/pom.xml [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.4</version>
```
Comment (kwsutter) [inline api/pom.xml]: 1.0.5

File: pom.xml [hunk 2]
```diff
@@ -17,17 +17,29 @@
...
-    <artifactId>jakarta.xml.rpc-api</artifactId>
+
+    <groupId>org.eclipse.ee4j.xml.rpc-api</groupId>
+    <artifactId>xml.rpc-api-parent</artifactId>
+    <packaging>pom</packaging>
     <version>1.1.3-SNAPSHOT</version>
-    
+    <name>jax-rpc-api</name>
+    <description>Eclipse Project for Stable EE4J APIs</description>
+    <url>https://github.com/eclipse-ee4j/xml.rpc-api</url>
```
Comment (kwsutter) [inline pom.xml]: jax-rpc-api

File: spec/src/main/asciidoc/xml-rpc-spec.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) 2017, 2019 Contributors to the Eclipse Foundation
+//
+
+= Jakarta XML RPC 1.1
+:authors: Jakarta XML RPC Team, https://projects.eclipse.org/projects/ee4j.jakartaee-stable
+:email: https://dev.eclipse.org/mailman/listinfo/xml.rpc-api-dev
```
Comment (kwsutter) [inline spec/src/main/asciidoc/xml-rpc-spec.adoc]: jakartaee-stable-dev

**Comment by dblevins** [comment]
@kwsutter I'm going to focus on the specs that seem to still not yet work with the generator.  If you want to take care of the tweaks, great, otherwise I'll come back and do it manually once all the PRs have been created.

**Comment by dblevins** [comment]
@kwsutter hold off on any tweaks.  I think it might be easiest for me to fix the generator and resubmit this with the other 3 pending PRs.

**Pull request merged**